db/migrate/20191213163728_add_arf_gi_bills.rb

```diff
@@ -0,0 +1,15 @@
+class AddArfGiBills < ActiveRecord::Migration[5.2]
+  def change
+    unless table_exists?("arf_gibills")
```

Comment:
Get rid of this.  See other comment below.

Reply:
Reverted commit to add this